packages/gatsby-plugin-catch-links/README.md

```diff
@@ -1,24 +1,33 @@
 # gatsby-plugin-catch-links
 
-Intercepts local links from markdown and other non-react pages and does a
-client-side pushState to avoid the browser having to refresh the page.
+This plugin intercepts all local links that have not been created in React using [`gatsby-link`](https://gatsbyjs.org/docs/gatsby-link), and replaces their behaviour with that of the `gatsby-link` [`navigate`](https://gatsbyjs.org/docs/gatsby-link/#programmatic-navigation). This avoids the browser having to refresh the whole page when navigating between local pages, preserving the Single Page Application (SPA) feel.
 
-For instance, in a markdown file with relative links (transformed
-to `a` tags by
-[`gatsby-transformer-remark`](/packages/gatsby-transformer-remark/)), this
-plugin replaces the default link behaviour
-with that of [`gatsby-link`'s `navigate`](https://gatsbyjs.org/docs/gatsby-link/#programmatic-navigation), preserving the
-SPA-like page change without reload.
+Example use cases:
 
-Check out the [_Using Remark_ example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-remark) to see this plugin in action.
+- A markdown file with relative links (transformed
+  to `a` tags by
+  [`gatsby-transformer-remark`](/packages/gatsby-transformer-remark/))
+- An `a` tag that has been created by a Content Management System (CMS) WYSIWYG editor
+
+## Installation
+
+```
+npm install --save gatsby-plugin-catch-links
+```
 
-## Install
+or
 
-`npm install --save gatsby-plugin-catch-links`
+```
+yarn add gatsby-plugin-catch-links
+```
 
 ## How to use
 
 ```javascript
 // In your gatsby-config.js
 plugins: [`gatsby-plugin-catch-links`]
 ```
+
+## Examples
+
+Check out the [_Using Remark_ example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-remark) to see this plugin in action.
```

Comment:
Maybe we could also link to https://using-remark.gatsbyjs.org/copy-linked-files-intercepting-local-links/#intercepting-local-links to showcase this in action?

Reply:
Sure, thats a good idea @pieh - I've just pushed another commit.

Reply:
Hey @pieh, are there any other changes you would like done?

Reply:
Hey, sorry for radio silence - I just added small comment